src/main/flight/pid.c

```diff
@@ -224,6 +225,11 @@ static FAST_RAM_ZERO_INIT uint8_t itermRelaxType;
 static FAST_RAM_ZERO_INIT uint8_t itermRelaxCutoff;
 #endif
 
+#if defined(USE_TBH)
+static FAST_RAM_ZERO_INIT float iPrevTBH[XYZ_AXIS_COUNT];
+static FAST_RAM_ZERO_INIT float errorSignPrevTBH[XYZ_AXIS_COUNT];
```

Comment:
`int8_t` or 'bool',  would be enough and save some ram
(`bool thisSign  = itermErrorRate > 0`)